ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/UpdatableDataColumnSidecarCustody.java

```diff
@@ -16,11 +16,15 @@
 import tech.pegasys.teku.infrastructure.async.SafeFuture;
 import tech.pegasys.teku.infrastructure.async.stream.AsyncStream;
 import tech.pegasys.teku.spec.datastructures.blobs.versions.fulu.DataColumnSidecar;
+import tech.pegasys.teku.spec.datastructures.blocks.SlotAndBlockRoot;
 import tech.pegasys.teku.spec.datastructures.util.DataColumnSlotAndIdentifier;
 
 public interface UpdatableDataColumnSidecarCustody extends DataColumnSidecarCustody {
 
   SafeFuture<Void> onNewValidatedDataColumnSidecar(DataColumnSidecar dataColumnSidecar);
 
   AsyncStream<DataColumnSlotAndIdentifier> retrieveMissingColumns();
+
+  /** Includes non-canonical sidecar identificators */
+  AsyncStream<DataColumnSlotAndIdentifier> retrieveMissingColumns(SlotAndBlockRoot blockId);
```

Comment:
Sorry, didn't find where this method is used

Reply:
Good catch, it was previous version, I don't need it anymore and all other changes in implementations